static/app/components/events/interfaces/crons/cronTimelineSection.tsx

```diff
@@ -120,6 +136,11 @@ export function CronTimelineSection({event, organization}: Props) {
   );
 }
 
+const MonitorLinkButton = styled(LinkButton)`
+  height: 26px;
```

Comment:
Damn, cc @vuluongj20

Reply:
Ye not feeling super confident about stuff like this, just matched it up with the resolution selector height for now

Reply:
We can use this — it's called `form` but it applies to both form inputs & buttons:
https://github.com/getsentry/sentry/blob/67bab6bd67f2aa3454b9f274ac1783494950b31e/static/app/utils/theme.tsx#L807-L830

Reply:
also I thought `LinkButton` had a size prop?

Reply:
oh lol my bad for some reason thought the next smallest size from "sm" was "zero" so I styled separately, will just change to a "xs" sized button which should fix the text + height